components/core/CMakeLists.txt

```diff
@@ -72,7 +72,15 @@ if (CLP_USE_STATIC_LIBS)
     if (APPLE)
         set(CLP_STATIC_LIBS_UNSUPPORTED_PLATFORM "macOS")
     elseif (EXISTS "/etc/centos-release")
-        set(CLP_STATIC_LIBS_UNSUPPORTED_PLATFORM "CentOS")
+        # NOTE:
+        # 1. We don't support static linking on any CentOS-based distro except manylinux 2.28 (which
```

Comment:
we should change `manylinux 2.28` to `manylinux_2_28` because `manylinux 2.28` is not a distribution + version. 2_28 refers to the glibc version iirc, not the version of many linux.